.env.example

```
@@ -89,3 +89,30 @@ CF_DNS_API_TOKEN=
+# Shown as the Reply-To / footer contact address.
+RAVEN_SUPPORT_ADDRESS=support@ravencloak.io
+# Public URL of the one-click unsubscribe endpoint (proxied by the API server).
+RAVEN_UNSUBSCRIBE_BASE_URL=https://api.ravencloak.io/api/v1/notifications/unsubscribe
+# HMAC-SHA256 secret for signing unsubscribe tokens (>= 32 random bytes).
+# Generate with: openssl rand -hex 32
+UNSUBSCRIBE_TOKEN_SECRET=
+# AI worker FastAPI base URL (POST /internal/summarize).
+RAVEN_AI_WORKER_HTTP_URL=http://ai-worker:8090
+# Anthropic API key used by the AI worker for summary generation.
+# BYOK — never commit. Claude Haiku is recommended for cost/latency.
+ANTHROPIC_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The ANTHROPIC_API_KEY key is duplicated

(DuplicatedKey)